packages/react-components/react-popover/stories/Popover/PopoverAnchorToCustomTarget.stories.tsx

```diff
@@ -24,16 +32,17 @@ const ExampleContent = () => {
 };
 
 export const AnchorToCustomTarget = () => {
-  const buttonRef = React.useRef<HTMLButtonElement>(null);
-  const positioningRef = React.useRef<PositioningImperativeRef>(null);
-  const styles = useStyles();
-
-  React.useEffect(() => {
+  const updateTarget = React.useCallback(() => {
     if (buttonRef.current) {
       positioningRef.current?.setTarget(buttonRef.current);
     }
   }, [buttonRef, positioningRef]);
```

Comment:
```suggestion
  const positioningRef = React.useRef<PositioningImperativeRef>(null);
  const buttonRef = React.useCallback((el: HTMLButtonElement | null) => {
    positioningRef.current?.setTarget(el)
  }, [positioningRef]);
```

React accepts functions as refs, that should enough 🐱